packages/palette/src/elements/PhoneInput/PhoneInput.tsx

```diff
@@ -445,6 +445,7 @@ const SelectOption = styled(Box)<{ selected?: boolean }>`
 
 const StyledInput = styled.input`
```

Comment:
Wondering why we didn't just use a Palette input here so that these don't drift. Alternatively could have extracted a mixin. This will probably crop up again as a problem area.